### PR TITLE
Patch - documentation updates

### PR DESCRIPTION
**Pull request by phidauex**
I've updated the documentation based on my experiences updating, and adding some clarifications and tips to address common problems that importers have been having. Hope it helps!

### CODE REVIEW

File: workflow.md [hunk 1]
```diff
@@ -56,9 +57,18 @@ To contribute to this project, you need to use the JOSM editor.  Here are some r
...
 
+### Pre-checking the data before merging
+
+The overall workflow is to "merge" the DRCOG data into your OSM Data Layer, while reviewing both data layers for possible conflicts. Some quality checks before merging will make things easier.
+
+* With the .osm file layer activated (no other OSM data visible), run the **JOSM Validator** (SHIFT+V) to look for errors. Common errors at this stage can include multiple buildings with the same address (usually an error where a house and a garage or shed have been given the same address tags). Refer to the section below on **fixing common errors** for solutions and guidelines.
+* While still in the new .osm file layer, **Select All** (CTRL+A) to select all objects, and then **Deselect Nodes** (SHIFT+U) to select all of the buildings. In the Selection window on the right side of the screen, scroll through the list of selected buildings to look for malformed street names. Examples of bad street names include:
+ * unexpanded abbreviations (Ct instead of Court, or Ln instead of Lane)
+ * use of the directional prefix "North" (not used in City and County of Denver)
```
Comment (frizatch) [inline workflow.md]: (not used in City and County of Denver *yet* so they aren't reflected on the street signs)

Comment (frizatch) [inline workflow.md]: Need to delete line 67 from this pull request b/c North can stay.

**Review by frizatch — COMMENTED** [review]
phidauex edits/additions are good! I just had one comment about the "North" in the street signs because we know Denver IS eventually going to have that information out in the world, but it's only in their dataset and not on the street signs yet... we can remove this instruction.

**Comment by phidauex** [comment]
Thanks, I removed the note about North streets. I'm sure you've discussed this, but leaving them in will cause a period of consternation with local mappers - it will trigger mass Osmose errors, and I'm sure a whole round of discussions about bad imports and "on the ground" mapping... If the decision is final from the City, then maybe a whole "fix the road naming" project will have to be started in parallel.

File: workflow.md [hunk 2]
```diff
@@ -61,68 +70,94 @@ To contribute to this project, you need to use the JOSM editor.  Here are some r
...
- * unexpanded abbreviations (Ct instead of Court, or Ln instead of Lane)
- * use of the directional prefix "North" (not used in City and County of Denver)
- * occasional errors in the automatic import, such as the street name *Steele Street* being expanded to *Streeteele Street*
+1. With the .osm file layer activated (no other OSM map data visible), run the **JOSM Validator** (SHIFT+V) to look for errors. Common errors at this stage can include multiple buildings with the same address (usually an error where a house and a garage or shed have been given the same address tags). Refer to the section below on **fixing common errors** for solutions and guidelines.
+1. While still in the new .osm file layer, **Select All** (CTRL+A) to select all objects, and then **Deselect Nodes** (SHIFT+U) to select all of the buildings. In the Selection window on the right side of the screen, scroll through the list of selected buildings to look for malformed street names. Examples of bad street names include:
+   * unexpanded abbreviations (Ct instead of Court, or Ln instead of Lane)
+   * use of the directional prefix "North" (not used in City and County of Denver)
```
Comment (frizatch) [inline workflow.md]: Comment the same as before, but it's now line 76! We can delete this because "North" is fine.

Comment (phidauex) [reply workflow.md]: Dang, forgot that I referenced north streets twice in there. Should be clean now.

**Review by geochasm — APPROVED** [review]
These changes add good additional detail; need to continue to review with respect to the "North" signage change happening in Denver.

**Review by frizatch — COMMENTED** [review]
Awesomesauce by phidauex! Much of this is format changing for readability, but the added sections on "Is it a building?" and making sweeping changes to the .osm file in an editor for street address (for example) is good value. Just the one line about removing "North" from the addresses is all I would adjust.

**Pull request merged**